website/docs/r/ses_domain_mail_from.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "aws"
+page_title: "AWS: ses_domain_mail_from"
+sidebar_current: "docs-aws-resource-ses-domain-mail-from"
+description: |-
+  Provides an SES domain MAIL FROM resource
+---
+
+# aws\_ses\_domain\_dkim
```

Comment:
Copypasta 🍝 : should be `aws_ses_domain_mail_from` (backslashes are no longer required in the documentation)